src/api/event/handler/chart/chart.go

```diff
@@ -1,28 +1,39 @@
-package notification
```

Comment:
Seems all handlers under the package `src/notifier/handler/notification` are webhook related? If they are, how about moving all of then into a separated package?

Reply:
only http and slack are left, all the others are move into ./api/event/handlers